swagger/public_v1/component_schemas/ProviderAttributes.yml

```diff
@@ -0,0 +1,119 @@
+---
+type: object
+required:
+  - code
+  - name
+properties:
+  accredited_body?:
+    type: boolean
+    description: "Is this provider an accredited body for other provider's courses."
+    example: "true"
+  city:
+    type: string
+    description: "Town or city"
+    example: "London"
+  county:
+    type: string
+    description: "County"
+    example: "London"
+  changed_at:
+    type: string
+    format: date-time
+    description: |
+      Date-time timestamp of when this provider or any of its related data changed.
+    example: "2019-06-13T10:44:31Z"
+  code:
+    type: string
+    description: |
+      Code that uniquely identifies this provider within a recruitment cycle.
+    maxLength: 3
+    minLength: 3
+    example: X99
+  created_at:
+    type: string
+    format: date-time
+    description: "Timestamp of when this provider was created."
+    example: "2019-06-13T10:44:31Z"
+  # latitude:
+  #   type: float
+  #   format:
+  #   description:
+  #   example:
+  # longitude:
+  #   type:
+  #   format:
+  #   description:
+  #   example:
+  postcode:
+    type: string
+    format: uk-postcode
+    description: "The provider's postcode"
+    maxLength: 8
+    example: SK2 6AA
+  name:
+    type: string
+    description: "The name of training provider."
+    example: "Long School"
+  provider_type:
+    type: string
+    description: "The type of provider."
+    example: "scitt"
+    enum:
+      - scitt
+      - lead_school
+      - university
+      - unknown
+      - invalid_value
+  recruitment_cycle_year:
+    type: string
```

Comment:
This should be an integer

Reply:
What do you think about keeping the potential open for this to be something that's not strictly an integer? "2020-2021" for example? I left it as a string just so we're not painted into a corner and it's not a large overhead.

Reply:
That would be a stealth breaking change, admittedly only if the client hadn't strictly followed the spec but 🤷‍♂ Leave it for now if you prefer.